### DISCUSSION TITLE
Add relationTypes for comments and replies

### DISCUSSION THREAD

**Discussion by KellyStathis** [datacite/datacite-suggestions#101] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

When registering DOIs for _responses to reviews_, there is not a clear choice of [relationType](https://datacite-metadata-schema.readthedocs.io/en/4/appendices/appendix-1/relationType/) to link the response to the review in the RelatedIdentifier field.

### What solution might meet your needs?

Add relationTypes "IsCommentOn"/"HasComment" and/or "IsReplyTo"/"HasReply", or similar. This would align with Crossref's relationship types.

### What alternatives have you tried or considered?

Using the relationType "References", which is not specific.

### Is there anything else you would like to share?

https://pidforum.org/t/commenting-reply-relation-types/3027/1

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [X] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by felixhelix** [comment]
I support this suggestion: we are to set up an open peer review pre-print server, where authors can (and should) reply to reviews: So far these resources have been hidden to the public, so that there was no need for a doi. But now that they are public and a) clearly an artefact in its own right and b) an important part of the review process, they should get a distinct DOI and be properly linked.
